.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+INSTRUCTIONS
+==========
+
+If you do not follow the below instructions, your issue may be dismissed.
+
+1. Search existing issues to avoid creating duplicates.
+
+2. If possible test using the latest release to make sure your issues has not already been fixed: 
+https://github.com/Azure/azure-iot-sdk-c/releases/latest 
+
+3. Do not share information from your Azure subscription here (connection strings, service names (IoTHub, Provisioning), etc...). Do so through your Azure subscription.
+
+4. Include enough information for us to address the bug, including a description of the bug, a Minimal Complete Reproducible Example that demonstrates the bug, and any relevant logs related to the bug.  
+
+For the MCVE, we recommend you If you are unsure how to enable logging, refer to this document: https://github.com/Azure/azure-iot-sdk-c/blob/master/doc/Iothub_sdk_options.md
+
+5. Delete these instructions before submitting the bug.
+
+------------------------------- delete above -------------------------------
+
+
+**Environment**
+Raspberry Pi, Raspbian Stretch Lite (Release 2018-11-13)
```

Comment:
Should we specify OS, toolchain etc instead of calling it environment?

Reply:
Yes that could be good. What do you think would be a relevant header for this one? Knowing what device they are running on and which OS they are running the SDK on are the first two relevant details that jump out at me. Should there be any other?

Reply:
The development machine, the OS of the device, compiler